quaternions/utils.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
```

Comment:
please, add the reference to the paper(s) you are implementing here.
Also, in the methods, there could be a line like `equation 2.3 from (1)`, or any other citation like this one.

Reply:
Still missing the reference paper to the sigma lerner calculation